dind/daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: kube-actions
+  name: binfmt-misc
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kube-actions
+      app.kubernetes.io/component: binfmt-misc
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kube-actions
+        app.kubernetes.io/component: binfmt-misc
+    spec:
+      initContainers:
+        - name: register
+          image: multiarch/qemu-user-static
+          args:
+            - --reset
+            - -p
+            - 'yes'
+          resources:
+            requests:
+              cpu: 2m
+              memory: 8Mi
+          securityContext:
+            privileged: true
+      containers:
+        - name: pause
+          image: k8s.gcr.io/pause
+          resources:
+            limits:
+              cpu: 2m
+              memory: 8Mi
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - linux
+                  - key: kubernetes.io/arch
+                    operator: In
+                    values:
+                      - amd64
```

Comment:
Is filterting by arch really necessary? I guess qemu will emulation will be useful if the hosts run on arm as well. qemu's setup will do the right thing either way: https://github.com/qemu/qemu/blob/master/scripts/qemu-binfmt-conf.sh#L292-L318.

Reply:
`qemu-static-user` is currently supported only on amd64 hosts: https://github.com/multiarch/qemu-user-static#supported-host-architectures